build.gradle

```diff
@@ -315,7 +315,7 @@ sonarqube {
     property "sonar.projectKey", "civil-service"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
     property "sonar.coverage.exclusions", "**/model/**, **/config/**/*Configuration.java, **/testingsupport/**, **/*ExternalTaskListener.java, **/*BaseExternalTaskHandler.java, **/stereotypes/**, **/*Exception.java, **/EventHistoryMapper*.java, **/model/hearingvalues/**, **/enums/hearing/**, **/fees/client/**, **/enums/sdo/**, **/service/PaymentsService.java"
-    property "sonar.cpd.exclusions", "**/*DocumentManagementService.java, **/*Spec*.java"
+    property "sonar.cpd.exclusions", "**/*DocumentManagementService.java, **/*Spec*.java, **/*CcdDashboardClaimantClaimMatcher.java"
```

Comment:
Do we need to add  to sonar exclusions?  Currently, CcdDashboardClaimMatcher is tested via the CcdClaimStatusDashboardFactoryTest.  We should add new tests here using the new matcher as well.  For now they can match the existing tests as the implementation for both matchers is the same.  But this means we won't forget to test when we make changes to the new matcher.

Reply:
@ElifDenizEsea has said that the exclusion is due to duplication rather than code coverage.  It would be worth moving the common implementations to a common base class, and then then we'd only need a test for each matcher separately when the implementation differs.  This could be done in one of the subsequent tickets if you prefer.